query-builder/src/main/java/com/datastax/oss/driver/internal/querybuilder/lhs/TupleLeftHandSide.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright DataStax, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.datastax.oss.driver.internal.querybuilder.lhs;
+
+import com.datastax.oss.driver.api.core.CqlIdentifier;
+import com.datastax.oss.driver.internal.querybuilder.CqlHelper;
+
+public class TupleLeftHandSide implements LeftHandSide {
```

Comment:
If this is only intended in multi-column relations, then I'd suggest renaming it to `MultiColumnLeftHandSide`.

Reply:
More generally speaking, "left operand" and "right operand" sound more conventional than "left hand side"  and "right hand side" and [this article](https://softwareengineering.stackexchange.com/questions/209922/formal-name-for-left-right-hand-side-of-an-expression) seems to confirm it.

Reply:
👍 to "multi-column" and "operand"